src/doc/man/cargo-logout.md

```diff
@@ -0,0 +1,55 @@
+# cargo-logout(1)
+
+## NAME
+
+cargo-logout --- Remove an API token from the registry locally
+
+## SYNOPSIS
+
+`cargo logout` [_options_]
+
+## DESCRIPTION
+
+This command will remove the API token from the local credential storage.
+Credentials are stored in `$CARGO_HOME/credentials.toml` where `$CARGO_HOME`
+defaults to `.cargo` in your home directory.
+
+If `--registry` is not specified, then the credentials for <https://crates.io/>
+will be removed.
+
```

Comment:
```suggestion
```

I think this should be removed if #11949 gets merged.

Reply:
Looks like this is now resolved